packages/colors-block/src/components/drops/DropsItem.tsx

```diff
@@ -87,50 +93,69 @@ export const DropsItem = ({ color, colorSpaces, isEditing, onBlur, onUpdate, onD
                 onBlur={onBlur}
             />
 
-            {colorSpaces?.map((colorSpaceId) => {
-                const mappedColorSpace = mapColorSpaces(colorSpaceId, color);
+            <div className={merge([colorSpaces.length > 0 && 'tw-mt-3'])}>
+                {colorSpaces?.map((colorSpaceId) => {
+                    const mappedColorSpace = mapColorSpaces(colorSpaceId, color);
 
-                return (
-                    <div
-                        data-test-id="color-space"
-                        key={colorSpaceId}
-                        className="tw-flex tw-items-center tw-mb-1 last:tw-mb-0"
-                    >
-                        <div className="tw-flex tw-items-center">
-                            <div className="tw-flex-[0_0_auto] tw-mr-1.5 tw-text-xs tw-text-black-70">
-                                {mappedColorSpace.label}
-                            </div>
+                    return (
+                        <div
+                            data-test-id="color-space"
+                            key={colorSpaceId}
+                            className={merge([
+                                'tw-flex tw-justify-center tw-items-center tw-mb-1 last:tw-mb-0',
+                                isEditing && 'tw-w-full',
+                                !isEditing && !mappedColorSpace.value && 'tw-mb-0',
```

Comment:
not sure if condition is still the same but would return alw a string
```suggestion
                                (isEditing || !mappedColorSpace.value) ? '' : 'tw-mb-0',
```